src/main.cpp

```diff
@@ -564,6 +564,10 @@ void event_on_configure(Root*, XEvent event) {
     }
 }
 
+void clientmessage(Root* root, XEvent* event) {
```

Comment:
Could be made `static`

Reply:
I think it does not matter much, because I'm planning to cut the `main.cpp` into smaller pieces by moving the Xlib-specific main-loop to an extra class.

Reply:
But I've made it static for the moment.